client/my-sites/customer-home/cards/tasks/style.scss

```diff
@@ -26,6 +26,18 @@
 		margin-bottom: 0;
 		order: 1;
 		align-self: center;
+
+		@include breakpoint( '<800px' ) {
+			display: none;
```

Comment:
If CSS is the approach being adopted here instead, could this check be removed?

https://github.com/Automattic/wp-calypso/blob/e03f5098b80a372efd26e2a5f7fe69c6772b5cbf/client/my-sites/customer-home/cards/tasks/task.jsx#L127

Reply:
Nice catch. I'm not sure why that was added there and not in the CSS. @mmtr Can you verify?

Reply:
It basically hid the illustration below 960px to prevent the issue being fixed here, but you need to refresh for it to take effect. There's a small amount of discussion about the trade-offs in terms of using that approach instead of CSS here: https://github.com/Automattic/wp-calypso/pull/40543#discussion_r400334196

For what it's worth, I think that the pattern in Customer Home is not to use CSS for this: https://github.com/Automattic/wp-calypso/search?q=path%3Aclient%2Fmy-sites%2Fcustomer-home+isdesktop&unscoped_q=path%3Aclient%2Fmy-sites%2Fcustomer-home+isdesktop

However, it would be necessary in this PR: since the illustration is being made to appear between 400px and 660px here, `isDesktop()` can't be used, though `isWithinBreakpoint`  may work. :)

Reply:
@Aurorum is right.

We prefer using the JS viewports utils (`isDesktop`, `isMobile`, `isWithinBreakpoint`) because it slightly improves the performance when something is not displayed. With a CSS breakpoint, the image is not displayed but still loaded, whereas JS viewports utils avoid the image from being rendered, so it is not loaded (less data to download).

As pointed out, the main downside is that JS viewports utils don't re-evaluate the screen size when resizing the browser window, so you need to reload the page to see the effects. This can be solved with more utils (`subscribeIsDesktop`, `subscribeIsMobile`, `subscribeIsWithinBreakpoint`) so you can re-render the component when the viewport changes. This comes with a small performance degradation (resizing listeners in JS can be expensive). However, final users don't tend to resize their browsers so we don't usually use these listeners. It's a bit harder for testing (requires to reload pages), but the performance benefits end up being worth it.

Reply:
@mmtr Ok, no worries. I'm going to just go ahead close this PR then.